sphinx/texinputs/make.bat_t

```diff
@@ -0,0 +1,55 @@
+@ECHO OFF
+
+REM Command file for Sphinx documentation
+
+pushd %~dp0
+
+{% if latex_engine == 'platex' -%}
+REM latexmkrc is read then overridden by latexmkjarc
+set PDFLATEX=latexmk -r latexmkjarc -pdfdvi -dvi- -ps-
+{% else -%}
+set PDFLATEX=latexmk -pdf -dvi- -ps-
+{% endif %}
+set LATEXOPTS=
```

Comment:
actually this means "remove LATEXOPTS envver".
refs: https://ss64.com/nt/set.html "Delete variable" section.

I think `set LATEXOPTS= ` or `set "LATEXOPTS= "` is expected one.

Reply:
done